### PR TITLE
Tweak CI build settings, license file, and README

### PR DESCRIPTION
**Pull request by seratch**
### Type of change (place an x in the [ ] that applies)

- [ ] New sample
- [ ] New feature
- [ ] Bug fix
- [x] Documentation

### Summary

This pull request improves a few files in the same way with https://github.com/slack-samples/deno-starter-template/pull/14

### Requirements (place an x in each [ ] that applies)

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

File: import_map.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "imports": {
     "deno-slack-sdk/": "https://deno.land/x/deno_slack_sdk@1.1.2/",
-    "deno-slack-api/": "https://deno.land/x/deno_slack_api@1.0.1/"
+    "deno-slack-api/": "https://deno.land/x/deno_slack_api@1.1.0/"
```
Comment (seratch) [inline import_map.json:4]: Upgraded to the latest

**Review by misscoded — APPROVED** [review]

File: .slack/apps.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (misscoded) [inline .slack/apps.json:1]: I don't think we want this checked in, do we?

**Review by misscoded — CHANGES_REQUESTED** [review]

File: .slack/apps.json (hunk 2 above)
Comment (seratch) [reply .slack/apps.json:1]: @misscoded If that’s the case, we should change .gitignore for all templates. Should we?

Comment (seratch) [reply .slack/apps.json:1]: Not sure about the reason why we currently ignore only dev json files

**Review by filmaj — APPROVED** [review]

File: .slack/apps.json (hunk 2 above)
Comment (misscoded) [reply .slack/apps.json:1]: I swore I had a previous discussion with someone that indicated both shouldn't be checked in, but I defer. It's not a use case I've thought about at all.

**Review by misscoded — APPROVED** [review]
LGTM, assuming that we're in agreement about checking in apps.json.

File: .slack/apps.json (hunk 2 above)
Comment (seratch) [reply .slack/apps.json:1]: Thanks for the replies!

**Pull request merged**